changelog/add_x86_omf.dd

```diff
@@ -0,0 +1,4 @@
+Added architecture x86_omf for windows-dmd
+
+In windows dmd, an architecture x86_omf is enabled for
+the platform specs when explicitly specify --arch=x86_omf or --arch=x86.
```

Comment:
`the platform specs when explicitly specif_ied_ via --arch=x86_omf or --arch=x86.`

Also, no newline at the end of file